adapters/ix/ix.go

```diff
@@ -222,6 +224,37 @@ func (a *IxAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalReque
 	return bidderResponse, errs
 }
 
+func getMediaTypeForBid(bid openrtb2.Bid, impMediaTypeReq map[string]openrtb_ext.BidType) (openrtb_ext.BidType, error) {
+	switch bid.MType {
+	case openrtb2.MarkupBanner:
+		return openrtb_ext.BidTypeBanner, nil
+	case openrtb2.MarkupVideo:
+		return openrtb_ext.BidTypeVideo, nil
+	case openrtb2.MarkupAudio:
+		return openrtb_ext.BidTypeAudio, nil
+	case openrtb2.MarkupNative:
+		return openrtb_ext.BidTypeNative, nil
+	}
```

Comment:
There's a json test covering the `MarkupVideo` case. I'm not going to ask for coverage of all other media types since the code is super simple and will be moved up into PBS-Core logic soon, although I welcome opinions for how to handle the fallback case.